drivers/flash/spi_flash_at45.c

```diff
@@ -534,6 +553,26 @@ static int spi_flash_at45_init(const struct device *dev)
 		return -ENODEV;
 	}
 
+	if (dev_config->reset) {
+		if (gpio_pin_configure(dev_config->reset->port,
+					dev_config->reset->pin,
+					GPIO_OUTPUT_ACTIVE | dev_config->reset->dt_flags)) {
```

Comment:
```suggestion
		if (gpio_pin_configure_dt(dev_config->reset, GPIO_OUTPUT_ACTIVE)) {
```

Reply:
@mbolivar-nordic WOW! It's great, never used this helper before. Will also replace pin configure in my project.
Big thanks for this comment.